docs/contrib/Logger.rst

```diff
@@ -317,7 +324,10 @@ call the utl::error/warn with a Tool ID and message ID. For
 compatibility these are defaulted to 'UKN' and '0000' until they are
 updated.
 
-Do no use "puts", ie, print) errors in regression tests. The logger prints the error now.
+Do no use "puts", i.e, (print) errors in regression tests. Use the report
+function to ensure proper logging of the message.
+  Logger::report(const std::string& message,
+                  const Args&... args)
 
 Init floorplan, openroad/src, init floorplan, dbSta, resizer, and opendp
```

Comment:
This comment is outdated.

Reply:
I have deleted this section

Reply:
Here is another statement that seems redundant in it's historical context. hence removing it...

Init floorplan, openroad/src, init floorplan, dbSta, resizer, and opendp
have been updated to use the Logger if you need examples of how to
initialize and use it.